ui/modal/modalClaimCollectionAdd/internal/claimCollectionAdd/view.jsx

```diff
@@ -8,17 +8,15 @@ import * as COLS from 'constants/collections';
 import * as ICONS from 'constants/icons';
```

Comment:
:small_red_triangle: Canonical used when adding items to playlist via the modal

Is canonical ok?  I recall most parts of the code is expecting permanent.

<img width="1253" alt="image" src="https://user-images.githubusercontent.com/64950861/189851432-adf992f8-51da-4cd0-8672-c1583bfca503.png">